lib/Extension/PhpCodeSniffer/Model/PhpCodeSnifferProcess.php

```diff
@@ -138,7 +139,7 @@ private function runDiagnosticts(string $url, string $text, array $options = [])
                 && $exitCode !== self::EXIT_FOUND_NON_FIXABLE_ERRORS
                 && $exitCode !== self::EXIT_FILES_NEEDS_FIXING
             ) {
-                $this->logger->error(
+                throw new RuntimeException(
```

Comment:
cc @bart-jaskulski the exceptions are now handled (was a regression with the last release that caused them not to be)